naturblick/utils/Backend.swift

```diff
@@ -239,7 +242,7 @@ class Backend {
     }
     
     func imageId(mediaId: String) async throws -> [SpeciesResult] {
-        let url = URL(string: Configuration.backendUrl + "androidimageid?mediaId=\(mediaId)")
+        let url = URL(string: Configuration.backendUrl + "androidimageid?mediaId=\(mediaId)&speciesVersion=\(currentSpeciesVersion)")
```

Comment:
in der `routes` von playback heißt es `GET     /androidimageid                             controllers.AndroidImageIdController.imageId(mediaId: MediaId, x: Option[Float] = None, y: Option[Float] = None, size: Option[Float] = None, original: Option[MediaId] = None, speciesDbVersion: Int = 1)`   also `speciesDbVersion` ??